ansible/provision.yml

```diff
@@ -27,6 +27,14 @@
     - common
     - mongodb
 
+- name: Message queue
+  user: ubuntu
+  hosts: tag_type_mq:mq
```

Comment:
No idea, I copied the form from the others in this file, was hoping you knew... :confounded: